src/runtime/sys.py

```diff
@@ -7,8 +7,16 @@ def exit(error_code: i32):
 
     quit(error_code)
 
+#  >>> argv >>>
```

Comment:
I would use different signs than <<< and >>> because those are used for git conflict resolution, so it might be confusing.